### ISSUE TITLE
A11y_mssql for VSCode_Select top 1000 Results window_Screenreader: Control type is incorrect for "Save as Excel" Button 

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode-mssql#16931]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag**:#A11yTCS; #Win10; #DesktopApp; #BM_mssql for VSCode_Jan21; #WCAG1.3.1; #A11ySev2; #Benchmark; #mssql for VSCode; #NVDA;#JAWS;

**Environment Details:**
Application Name: mssql for VSCode
Operating System: Win10
Extension Version: 1.10.0

**Repro Steps:**

1. Open Visual studio code
2. Click on extension button and search for "mssql"
3. Click on install and move to the SQL Server button
4. Click on Add Connection Button and insert the valid credentials
5. After adding the server move to database expand it  and select System database expand then Master expand then ables expand and then System table expand and right click on it and select top 1000 option
6. Select top 1000 window will open navigate to Results window
7. Open NVDA as Screenreader and check the control type set for "Save as Excel" Button

**Actual:**
When we navigate to "Save as Excel" Button using  NVDA as Screenreader the control type is set as link

**Expected:**
When we navigate to "Save as Excel" Button using  NVDA as Screenreader the control type should be announced as Button

**User Impact:**
Speech to text users will get ambiguous information, as the name of the control is "Save as Excel" but the control is set as "Link".

**Comment by fsteffi** [comment]
Github Tags:#A11yMAS;

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro
Version: 1.76.0-insider (user setup)
![16931_issue fixed](https://user-images.githubusercontent.com/93374043/220058324-c07f8e2b-3b09-433c-8b81-f6f2f6275940.png)


**GithubTags**:#Closed;
